ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/PointSchedule/Tests.hs

```diff
@@ -121,7 +120,7 @@ instance QC.Arbitrary HeaderPointScheduleInput where
     branchTips <- genTipPoints
     let branchCount = length branchTips
         tpCount = sum $ map length branchTips
-    ts <- scanl1 (+) . sort <$> replicateM tpCount (chooseDiffTime (7, 12))
+    ts <- map Time <$> scanl1 (+) . sort <$> replicateM tpCount (chooseDiffTime (7, 12))
```

Comment:
If you use `Data.Coerce.coerce` here, you can do the conversion without computing a new list. e.g.
```
Data.Coerce.coerce <$> scanl1 (+) . sort <$> replicateM tpCount (chooseDiffTime (7, 12))
```